docs/api/browser-window.md

```diff
@@ -498,6 +498,14 @@ Returns:
 
 Emitted on 3-finger swipe. Possible directions are `up`, `right`, `down`, `left`.
 
+#### Event: 'begin-sheet' _macOS_
```

Comment:
I agree. I was not able to decide because there is also the pattern of `*-full-screen` events.